config/config.go

```diff
@@ -75,6 +75,29 @@ func UpdateNetclient(c Config) {
 	netclient = c
 }
 
+// UpdateHost - update host with data from server
+func UpdateHost(newHost *models.Host) {
+	netclient.Host.Name = newHost.Name
+	netclient.Host.Verbosity = newHost.Verbosity
+	netclient.Host.MTU = newHost.MTU
+	if newHost.ListenPort > 0 {
+		netclient.Host.ListenPort = newHost.ListenPort
+	}
+	if newHost.ProxyListenPort > 0 {
+		netclient.Host.ProxyListenPort = newHost.ProxyListenPort
+	}
+	netclient.Host.IsDefault = newHost.IsDefault
+	netclient.Host.DefaultInterface = newHost.DefaultInterface
+	if !netclient.Host.ProxyEnabledSet {
+		netclient.Host.ProxyEnabled = newHost.ProxyEnabled
+		netclient.Host.ProxyEnabledSet = true
```

Comment:
this if block `if !netclient.Host.ProxyEnabledSet` is not clear to me, what is the intention of this?

Reply:
This is to prevent a second server from changing the value it it was set by a previous server